pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -874,16 +895,16 @@ Every continuation @racket[_C] can be partitioned into
 @deftech{continuation frames} @frame[1], @frame[2], ..., @frame["n"]
 such that @racket[_C] = @*sub[@frame[1] @*sub[@frame[2] @*sub["..."
 @frame["n"]]]], and no frame @frame["i"] can be itself partitioned
-into smaller continuations. Evaluation steps add and remove frames to
+into smaller continuations. Evaluation steps add frames to and remove frames from
 the current continuation, typically one at a time.
 
 Each frame is conceptually annotated with a set of
-@deftech{continuation marks}. A mark consists of a key and its value;
-the key is an arbitrary value, and each frame includes at most one
-mark for any key. Various operations set and extract marks from
+@deftech{continuation mark}s. A mark consists of a key and its value.
```

Comment:
Its fine to keep the `s` inside the `deftech`

Reply:
Okay.